frontend/config-assistant/src/model/OneOfAnyOfSelectionOption.ts

```diff
@@ -22,15 +22,16 @@ export class OneOfAnyOfSelectionOption {
  * @param index  the index of the schema in the list of schemas
  */
 export function schemaOptionToString(schema: JsonSchema, index: number): string {
-  let result = `${index}`;
-
   if (schema.title) {
-    result += ': ' + schema.title;
-  } else if (schema.description) {
-    result += ': ' + schema.description;
-  } else if (schema.type) {
-    result += ': ' + schema.type;
```

Comment:
what text will be shown for sub-schema that do neither have title nor description?

Specifically, when a schema has multiple types, our pre-processing will convert those types into oneOf. Each oneOf sub-schema will contain nothing but the type. Also, independent of types to oneOf conversion, it can happen that a sub-schema contains only a type.

Reply:
For example, if you choose the example schema and remove the title property:
![image](https://github.com/PaulBredl/config-assistant/assets/34026908/3c526544-c37f-461d-9e44-ec7937214325)


Problematic are the big oneOfs, but I don't think there is a good solution for that. As a guideline we can say that schema authors should give titles to oneOf options